drivers/sensor/CMakeLists.txt

```
@@ -68,6 +68,7 @@ add_subdirectory_ifdef(CONFIG_QDEC_NRFX		qdec_nrfx)
 add_subdirectory_ifdef(CONFIG_QDEC_SAM		qdec_sam)
 add_subdirectory_ifdef(CONFIG_TEMP_NRF5		nrf5)
 add_subdirectory_ifdef(CONFIG_SBS_GAUGE		sbs_gauge)
+add_subdirectory_ifdef(CONFIG_SCD30         scd30)
```

although we generally use spaces in CMake files, then this file seems to have everything aligned with tabs, so please follow that principle so it looks similar to all other lines:
```suggestion
add_subdirectory_ifdef(CONFIG_SCD30<tab><tab>scd30)
```

---

seems you added 3 tabs, not 2.

small note, please let the reviewer decide if the comment is resolved :wink:

---

Should be fixed now